stl/inc/future

```diff
@@ -1384,7 +1384,7 @@ public:
 
     [[noreturn]] _Fake_no_copy_callable_adapter(const _Fake_no_copy_callable_adapter& _Other)
         : _Storage(_STD move(_Other._Storage)) {
-        _CSTD abort(); // shouldn't be called, see GH-3888
+        _CSTD abort(); // shouldn't be called

```

Comment:
(  I find `"see GH-3888"` a bit awkward, as the reasons has been explained at the beginning of the class :|  )